modules/openxoutstreamBidAdapter.js

```diff
@@ -0,0 +1,225 @@
+import { config } from 'src/config';
+import { registerBidder } from 'src/adapters/bidderFactory';
+import * as utils from 'src/utils';
+import { BANNER } from 'src/mediaTypes';
+import { parse } from 'src/url';
+
+const SUPPORTED_AD_TYPES = [BANNER];
+const BIDDER_CODE = 'openxoutstream';
+const BIDDER_CONFIG = 'hb_pb_ym';
+const BIDDER_VERSION = '1.0.0';
+const CURRENCY = 'USD';
+const NET_REVENUE = true;
+const TIME_TO_LIVE = 300;
+const YM_SCRIPT = `!function(e,t){if(void 0===t._ym){var a=Math.round(5*Math.random()/3)+'';t._ym='';var m=e.createElement('script');m.type='text/javascript',m.async=!0,m.src='//static.yieldmo.com/ym.'+a+'.js',(e.getElementsByTagName('head')[0]||e.getElementsByTagName('body')[0]).appendChild(m)}else t._ym instanceof String||void 0===t._ym.chkPls||t._ym.chkPls()}(document,window);`;
+const PLACEMENT_ID = '1986307928000988495';
```

Comment:
do we want to use a different placement id from the openx adapter so that we can tell them apart?

Reply:
@yieldmo-rao ^

Reply:
I've internally asked the question - awaiting confirmation.

Reply:
@HolzAndrew , here's the new CR ID: `2052941939925262540`

Nice catch @merrdy  :)

Reply:
ok I've updated.

Reply:
Perfect! Thanks!